src/vcFBX.cpp

```diff
@@ -804,8 +806,43 @@ void vcFBX_Close(vdkConvertCustomItem *pConvertInput)
     if (pFBX->pTrivox)
       vdkTriangleVoxelizer_Destroy(&pFBX->pTrivox);
 
+    if (pFBX->pScene != nullptr)
+    {
+      pFBX->pScene->Destroy(true);
+      pFBX->pScene = nullptr;
+    }
+
     if (pFBX->pManager != nullptr)
+    {
       pFBX->pManager->Destroy();
+      pFBX->pManager = nullptr;
+    }
+
+    pFBX->uvQueue.Deinit();
+
+    vcFBX_CleanMaterials(&pFBX->materials);
+  }
+}
+
+void vcFBX_Destroy(vdkConvertCustomItem* pConvertInput)
```

Comment:
`vcFBX_Destroy()` will be called after `vcFBX_Close` always. The only thing that should be be cleaned up here is anything allocated inside `vcFBX_AddItem()`.

Checking `pConvertInput->pData != nullptr` is also unnecessary.